godtools/Views/Buttons/GTButton.swift

```diff
@@ -47,10 +47,28 @@ class GTButton: UIButton {
     
     @IBInspectable var translationKey: String = "" {
         didSet {
-            self.setTitle(translationKey.localized, for: .normal)
+            let state = self.isEnabled ? UIControlState.normal : UIControlState.disabled
+            
+            self.setTitle(translationKey.localized, for: state)
         }
     }
     
+    func designAsUnavailableButton() {
+        self.designAsToolDetailButton()
+        
+        self.color = .gtWhite
+        self.tintColor = .gtWhite
+        self.borderWidth = 0.0
+        self.borderColor = .clear
+        self.backgroundColor = .gtGrey
+        self.setImage(#imageLiteral(resourceName: "download_white"), for: .normal)
+        self.translationKey = "unavailable"
+        
+        self.isEnabled = false
```

Comment:
I believe this line should be before self.translationKey = "unavailable", because of the `didSet` of the translationKey